### PR TITLE
Fix version checker tests

### PR DESCRIPTION
**Pull request by xoriole**
This PR fixes broken version checker tests in the devel branch

### CODE REVIEW

**Comment by xoriole** [comment]
Windows tests failed. Unrelated to version checker but could be because of the port issue. Will retry to see if it persists.
```
=========================== short test summary info ===========================
FAILED src/tribler-core/tribler_core/modules/tunnel/tests/test_triblertunnel_community.py::TestTriblerTunnelCommunity::test_perform_http_request_multipart
=========== 1 failed, 612 passed, 5 skipped, 736 warnings in 31.50s ===========
```
```
================================== FAILURES ===================================
_______ TestTriblerTunnelCommunity.test_perform_http_request_multipart ________
[gw10] win32 -- Python 3.8.4 c:\users\tribler\appdata\local\programs\python\python38\python.exe

self = <test_triblertunnel_community.TestTriblerTunnelCommunity testMethod=test_perform_http_request_multipart>

    async def test_perform_http_request_multipart(self):
        """
        Test whether getting a large HTTP response works
        """
        self.add_node_to_experiment(self.create_node())
        self.nodes[1].overlay.settings.peer_flags.add(PEER_FLAG_EXIT_HTTP)
        await self.introduce_nodes()
    
        http_port = get_random_port()
        http_tracker = HTTPTracker(http_port)
        http_tracker.tracker_info.add_info_about_infohash('0', 0, 0)
        http_tracker.tracker_info.infohashes['0']['downloaded'] = os.urandom(10000)
        await http_tracker.start()
        response = await self.nodes[0].overlay.perform_http_request(('127.0.0.1', http_tracker.port),
                                                                    b'GET /scrape?info_hash=0 HTTP/1.1\r\n\r\n')
    
>       self.assertEqual(response.split(b'\r\n')[0], b'HTTP/1.1 200 OK')
E       AttributeError: 'NoneType' object has no attribute 'split'

src\tribler-core\tribler_core\modules\tunnel\tests\test_triblertunnel_community.py:600: AttributeError
```

**Comment by xoriole** [comment]
retry this please

**Comment by xoriole** [comment]
retest this please

File: src/tribler-core/tribler_core/modules/tests/test_versioncheck.py [hunk 1]
```diff
@@ -108,30 +109,34 @@ async def test_non_json_response(version_check_manager, version_server):
...
+    response = json.dumps({'name': NEW_VERSION_ID})
+
+    import tribler_core.modules.versioncheck_manager as vcm
+    old_timeout = vcm.VERSION_CHECK_TIMEOUT
+    vcm.VERSION_CHECK_TIMEOUT = 0.001
+
+    has_new_version = await version_check_manager.check_new_version()
+    assert not has_new_version
+
+    vcm.VERSION_CHECK_TIMEOUT = old_timeout
```
Comment (devos50) [inline src/tribler-core/tribler_core/modules/tests/test_versioncheck.py:124]: It seems that there is a test for the timeout of the version check, but the code coverage reporter shows an uncovered line there?

Comment (ichorid) [reply src/tribler-core/tribler_core/modules/tests/test_versioncheck.py:124]: Yep, we *definitely* want to cover *that* line.

**Review by ichorid — CHANGES_REQUESTED** [review]
Please, cover the remaining line. Can't be too safe with this stuff.

File: src/tribler-core/tribler_core/modules/tests/test_versioncheck.py (hunk 1 above)
Comment (xoriole) [reply src/tribler-core/tribler_core/modules/tests/test_versioncheck.py:127]: @devos50 @ichorid to address this I have now added a separate test to cover the timeout case for individual API check

**Review by ichorid — APPROVED** [review]

**Pull request merged**